docker-compose-dev-e2e.yaml

```diff
@@ -34,67 +44,76 @@ services:
-  #   restart: always
-  #   ports:
-  #     - "5432:5432"
-  #   environment:
-  #     - POSTGRES_USER=postgres
-  #     - POSTGRES_PASSWORD=postgres
-
   #backends
-  # login-backend:
-  #   container_name: login-backend
-  #   image: ghcr.io/gamify-it/login-backend:main
-  #   restart: always
-  #   pull_policy: always
-  #   expose:
-  #     - "80"
-  #   environment:
-  #     - JWT_KEY=SecretKEY
-  #     - JWT_VALIDITY_DURATION=720h
-  #     - POSTGRES_URL=postgresql://postgres:postgres@login-db:5432/
+  keycloak:
+    container_name: keycloak
+    expose:
+      - "80"
+    image: quay.io/keycloak/keycloak:19.0
+    command:
+      - "start-dev"
+    environment:
+      KEYCLOAK_ADMIN: admin
+      KEYCLOAK_ADMIN_PASSWORD: admin
+      KC_DB: postgres
+      KC_DB_URL: jdbc:postgresql://keycloak-db:5432/postgres
+      KC_DB_USERNAME: postgres
+      KC_DB_PASSWORD: postgres
+      KC_HOSTNAME_STRICT: false
+      KC_PROXY: edge
+      KC_HTTP_RELATIVE_PATH: /keycloak
+      KC_HTTP_PORT: 80
 
-  # overworld-backend:
-  #   container_name:   overworld-backend
-  #   image: ghcr.io/gamify-it/overworld-backend:main
-  #   restart: always
-  #   pull_policy: always
-  #   expose:
-  #     - "80"
-  #   depends_on:
-  #     - overworld-db
-  #   environment:
-  #     - POSTGRES_USER=postgres
-  #     - POSTGRES_PASSWORD=postgres
-  #     - POSTGRES_URL=postgresql://overworld-db:5432/postgres
+  overworld-backend:
+    container_name: overworld-backend
+    image: ghcr.io/gamify-it/overworld-backend:main
+    restart: always
+    pull_policy: always
+    expose:
+      - "80"
+    depends_on:
+      - overworld-db
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_URL=postgresql://overworld-db:5432/postgres
+      - REGEXGAME_URL=http://host.docker.internal:8080/api/v1
+      - KEYCLOAK_ISSUER=http://localhost/keycloak/realms/Gamify-IT
+      - KEYCLOAK_URL=http://keycloak/keycloak/realms/Gamify-IT
 
   #frontends
-  # login-frontend:
-  #   container_name: login-frontend
-  #   image: ghcr.io/gamify-it/login-frontend:main
-  #   restart: always
-  #   pull_policy: always
-  #   expose:
-  #     - "80"
+  landing-page:
+    container_name: landing-page
+    image: ghcr.io/gamify-it/landing-page:main
+    restart: always
+    pull_policy: always
+    expose:
+      - "80"
 
-  # overworld:
-  #   container_name: overworld
-  #   image: ghcr.io/gamify-it/overworld:main
-  #   restart: always
-  #   pull_policy: always
-  #   expose:
-  #     - "80"
+  overworld:
+    container_name: overworld
+    image: ghcr.io/gamify-it/overworld:main
+    restart: always
+    pull_policy: always
+    expose:
+      - "80"
 
-  # lecturer-interface:
-  #   container_name: lecturer-interface
-  #   image: ghcr.io/gamify-it/lecturer-interface:main
-  #   restart: always
-  #   pull_policy: always
-  #   expose:
-  #     - "80"
+  regex-game:
```

Comment:
Why are we inserting the `regex-game` here?
Isn't the `e2e` config supposed to not start it but rather to pick up a started development/debugging instance running on the host?